fix board selection and workspace input dialogs width and height

AlbyIanna opened the pull request:
### Motivation
Some dialogs have unexpected behaviours as described in #1350 

### Change description
fixed width and height of workspace input dialog and board selection dialog

### Other information
Closes #1350

### Reviewer checklist

* [ ] PR addresses a single concern.
* [ ] The PR has no duplicates (please search among the [Pull Requests](https://github.com/arduino/arduino-ide/pulls) before creating one)
* [ ] PR title and description are properly filled.
* [ ] Docs have been added / updated (for bug fixes / features)

kittaakos reviewed (changes requested):
It does not fix #1350. See why: https://github.com/arduino/arduino-ide/pull/1406#issuecomment-1247817487.

91volt commented:
Checked the build, the delete dialog is missing the fixed width:
<img width="1414" alt="Schermata 2022-09-19 alle 15 29 37" src="https://user-images.githubusercontent.com/92443068/191029224-6ac3e813-3c49-4268-99fd-86599af9307b.png">

Once it is fixed the PR is ok to me

kittaakos reviewed (commented):
The `New`, `Rename`, and `Delete` tabs have different sizes:
<img width="678" alt="Screen Shot 2022-09-19 at 18 05 28" src="https://user-images.githubusercontent.com/1405703/191063091-df2cdffc-4dc5-4ecf-81c8-76ec0c44774c.png">

<img width="680" alt="Screen Shot 2022-09-19 at 18 05 48" src="https://user-images.githubusercontent.com/1405703/191063166-bf41a787-41b6-4ae6-b072-229221998888.png">

<img width="683" alt="Screen Shot 2022-09-19 at 18 06 12" src="https://user-images.githubusercontent.com/1405703/191063211-9ac1a4b5-3566-4b86-8887-64cc5bed20e9.png">

----

The board list height changes when filtering in the list (https://github.com/arduino/arduino-ide/issues/1350#issuecomment-1227418516):

https://user-images.githubusercontent.com/1405703/191063290-86d8a1fa-4a52-4ffb-928f-999b6f3796b5.mp4

Should this PR cover https://github.com/arduino/arduino-ide/issues/1350#issuecomment-1248899194? If yes, let's make this [#1382](https://github.com/arduino/arduino-ide/pull/1382) dependent. You can also cherry-pick the `overflow` changes or copy them here.

<img width="658" alt="Screen Shot 2022-09-19 at 18 06 32" src="https://user-images.githubusercontent.com/1405703/191064293-e6bd547f-d800-43d6-93f3-f63e23a4636c.png">

AlbyIanna commented:
Thanks for the review @kittaakos and @91volt. I've fixed the remaining dialogs, now the min-size for _any_ dialog is set by default to 480px.

--- 

I've fixed the board list changing size when filtering, and took the liberty to change the UI when no board is found for the search made by to user: 
![image](https://user-images.githubusercontent.com/6939054/191192686-da3281b0-af30-4595-aeec-ee9161cb5a0b.png)
The alternative was to show a white empty rectangle, which wasn't very nice to me. @91volt do you like it? ☝️

AlbyIanna commented:
> Should this PR cover https://github.com/arduino/arduino-ide/issues/1350#issuecomment-1248899194? If yes, let's make this https://github.com/arduino/arduino-ide/pull/1382 dependent. You can also cherry-pick the overflow changes or copy them here.

@kittaakos since #1382 is already approved, I'm okay with waiting for it to be merged.

kittaakos commented:
> @kittaakos since #1382 is already approved, I'm okay with waiting for it to be merged.

Could you please rebase? #1382 has been merged. Your branch is three commits behind the `main`. Thank you!

<img width="1311" alt="Screen Shot 2022-09-20 at 12 02 58" src="https://user-images.githubusercontent.com/1405703/191229617-32169f03-f96b-4edf-9048-4705744172e8.png">

AlbyIanna commented:
> Could you please rebase? #1382 has been merged. Your branch is three commits behind the `main`. Thank you!

Done. I've also fixed the Additional Boards URL focus outline.

![image](https://user-images.githubusercontent.com/6939054/191239866-04d2be98-1cf8-452e-803b-0b2f8d525f38.png)

kittaakos reviewed (approved):
It looks great. Thank you!
